commands/runner.go

```diff
@@ -188,3 +198,26 @@ func splitAliasCmd(cmd string) ([]string, error) {
 
 	return words, nil
 }
+
+func getListOfBuiltInCommands() map[string]bool {
+	gitHelpOutput, _ := cmd.New("git help -a").CombinedOutput()
+	builtInCommands := parseGitHelpOutput(gitHelpOutput)
+	for hubCommand, _ := range CmdRunner.All() {
+		builtInCommands[hubCommand] = true
+	}
+	return builtInCommands
+}
+
+func parseGitHelpOutput(gitHelpOutput string) map[string]bool {
```

Comment:
Almost. It breaks a test in sync.feature and rightly so. We get list of branches using `branch --list` and trim the spaces on both ends before returning from gitOutput. We then do rev-parse on the list we get. If we don't trim the list, we end up checking for something like `refs/remotes/origin/  bugfix`. Adding trim to git.LocalBranches() instead of gitOutput fixes this.